python/py-anytree/Portfile

```diff
@@ -31,6 +31,8 @@ if {${name} ne ${subport}} {
     depends_lib-append \
                     port:py${python.version}-six
 
+    patchfiles          patch-setup.py.diff
```

Comment:
please align this in the same way as the rest of the Portfile (also in `py-bottlenose`)